src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGenerator.cs

```diff
@@ -449,7 +452,7 @@ private async Task<OpenApiOperation> GenerateOpenApiOperationFromMetadataAsync(A
                             content.Schema = new OpenApiSchema()
                             {
                                 AllOf = requestParameters.Select(s =>
-                                    GenerateSchemaIncludingFormFile(s, GenerateSchema(
+                                    GenerateSchemaIncludingFromFormWithNoReference(s, GenerateSchema(
```

Comment:
Not happy with the name, because now it's not only for IFromFile

Reply:
`GetSchemaForForm()`?

Reply:
GetSchemaIncludingForForm?